create_sdf_from_mesh.py

```diff
@@ -0,0 +1,441 @@
+        "--mu_static",
+        type=float,
+        default=None,
+        help="The coefficient of static friction.",
+    )
+    parser.add_argument(
+        "--resolution",
+        type=int,
+        default=10000000,
+        help="VHACD voxel resolution.",
+    )
+    parser.add_argument(
+        "--maxhulls",
+        type=int,
+        default=100,
+        help="VHACD maximum number of convex hulls/ mesh pieces.",
+    )
+    parser.add_argument(
+        "--minVolumePerCH",
+        type=float,
+        default=0.001,
+        help="VHACD minimum convex hull volume.",
+    )
+    parser.add_argument(
+        "--maxNumVerticesPerCH",
+        type=int,
+        default=64,
+        help="VHACD maximum number of triangles per convex hull.",
+    )
+    parser.add_argument(
+        "--pca",
+        action="store_true",
+        help="Whether to normalize the mesh with PCA before performing the VHACD "
+        + "convex decomposition.",
+    )
+    parser.add_argument(
+        "--preview",
+        action="store_true",
+        help="Whether to preview the decomposition.",
+    )
+    parser.add_argument(
+        "--log_level",
+        type=str,
+        default="INFO",
+        choices=["CRITICAL", "ERROR", "WARNING", "INFO", "DEBUG"],
+        help="Log level.",
+    )
+
+    args = parser.parse_args()
+    logging.basicConfig(level=args.log_level)
+
+    mesh_path = Path(os.path.abspath(args.mesh))
+    if not mesh_path.exists():
+        logging.error(f"No mesh found at {mesh_path}!")
+        sys.exit(1)
+
+    mass = args.mass
+    if mass <= 0:
+        logging.error(f"Got a mass of {mass} kg. Mass must be positive!")
+        sys.exit(1)
+
+    is_compliant = args.compliant
+    hydroelastic_modulus = args.hydroelastic_modulus
+    if (
+        is_compliant
+        and hydroelastic_modulus is not None
+        and hydroelastic_modulus < 0
+    ):
+        logging.error(
+            f"Got a Hydroelastic modulus of {hydroelastic_modulus:.3e} Pa. A "
+            + "non-negative value is required!"
+        )
+        sys.exit(1)
+
+    hunt_crossley_dissipation = args.hunt_crossley_dissipation
+    if hunt_crossley_dissipation is not None and hunt_crossley_dissipation < 0:
+        logging.error(
+            f"Got a Hunt-Crossley dissipation of {hunt_crossley_dissipation:.3f} s/m. A "
+            + "non-negative value is required!"
+        )
+        sys.exit(1)
+
+    mu_dynamic = args.mu_dynamic
+    if mu_dynamic is not None and mu_dynamic < 0:
+        logging.error(
+            f"Got a coefficient of dynamic friction of {mu_dynamic:.3f}. A "
+            + "non-negative value is required!"
+        )
+        sys.exit(1)
+
+    mu_static = args.mu_static
+    if mu_static is not None and mu_static < 0:
+        logging.error(
+            f"Got a coefficient of static friction of {mu_static:.3f}. A "
+            + "non-negative value is required!"
+        )
+        sys.exit(1)
+
+    create_sdf_from_mesh(
```

Comment:
can we add a python test, at least (if not a command line test)?  i don't need for it to cover every functionality, but want CI to at least make sure it actually runs, so that we can catch drake deprecations, and/or dependency issues/changes.

Reply:
I will finish this up later today